frontend/src/components/grantaccess/form/ForestClientInput.vue

```diff
@@ -0,0 +1,194 @@
+<script setup lang="ts">
+import { ref, watch } from 'vue';
+import { ErrorMessage, Field } from 'vee-validate';
+import InputText from 'primevue/inputtext';
+import ApiServiceFactory from '@/services/ApiServiceFactory';
+import { requireInjection } from '@/services/utils';
+import LoadingState from '@/store/LoadingState';
+import ForestClientCard from '@/components/grantaccess/ForestClientCard.vue';
+import { IconSize } from '@/enum/IconEnum';
+import {
+    FamForestClientStatusType,
+    type FamForestClient,
+} from 'fam-app-acsctl-api';
+
+const FOREST_CLIENT_INPUT_MAX_LENGTH = 8;
+const appActlApiService = requireInjection(
+    ApiServiceFactory.APP_ACCESS_CONTROL_API_SERVICE_KEY
+);
+
+const props = defineProps({
+    userId: { type: String, required: true },
+    roleId: String,
+    fieldId: { type: String, default: 'forestClientNumbers' }, // field id used in the form validation
+});
+
+const emit = defineEmits([
+    'setVerifiedForestClients',
+    'removeVerifiedForestClients',
+    'resetVerifiedForestClients',
```

Comment:
these three emit functions are used to set the "verifiedForestClients" data in the formData in parent component